selfdrive/monitoring/test_monitoring.py

```diff
@@ -5,19 +5,15 @@
 from cereal import car, log
 from common.realtime import DT_DMON
 from selfdrive.controls.lib.events import Events
-from selfdrive.monitoring.driver_monitor import DriverStatus, \
-                                  _AWARENESS_TIME, _AWARENESS_PRE_TIME_TILL_TERMINAL, \
-                                  _AWARENESS_PROMPT_TIME_TILL_TERMINAL, _DISTRACTED_TIME, \
-                                  _DISTRACTED_PRE_TIME_TILL_TERMINAL, _DISTRACTED_PROMPT_TIME_TILL_TERMINAL, \
-                                  _POSESTD_THRESHOLD, _HI_STD_FALLBACK_TIME
+from selfdrive.monitoring.driver_monitor import DriverStatus, DRIVER_MONITOR_SETTINGS as dm_settings
 
 EventName = car.CarEvent.EventName
 
-_TEST_TIMESPAN = 120  # seconds
-_DISTRACTED_SECONDS_TO_ORANGE = _DISTRACTED_TIME - _DISTRACTED_PROMPT_TIME_TILL_TERMINAL + 1
-_DISTRACTED_SECONDS_TO_RED = _DISTRACTED_TIME + 1
-_INVISIBLE_SECONDS_TO_ORANGE = _AWARENESS_TIME - _AWARENESS_PROMPT_TIME_TILL_TERMINAL + 1
-_INVISIBLE_SECONDS_TO_RED = _AWARENESS_TIME + 1
+TEST_TIMESPAN = 120  # seconds
+DISTRACTED_SECONDS_TO_ORANGE = dm_settings["_DISTRACTED_TIME"] - dm_settings["_DISTRACTED_PROMPT_TIME_TILL_TERMINAL"] + 1
```

Comment:
these super long names make it hard to read the tests. maybe something like `WARN_DT`?

Reply:
will do a names cleanup in another pr